ReactJS Lesson #5 Daniil Lyadnov

websterday-online opened the pull request.

idoleg commented on .gitignore:
```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# ignor file and folders
+.idea
```
Отмените эти изменений, конфликт

idoleg reviewed (changes requested):
Поправить конфликт и я прийму

websterday-online committed:
Update .gitignore

Конфликт из-за не правильно пути к папке или не правильно указал папу?
Просто я удалил свой .gitignore и оставил ваш, но мне нужно убрать .idea/ от phpstorm.

websterday-online committed:
Lesson #4

websterday-online committed:
Merge branch 'DaniilLyadnov' of github.com:websterday-online/react-01.06 into DaniilLyadnov

websterday-online committed:
Lesson #5

websterday-online committed:
Merge branch 'master' into DaniilLyadnov

websterday-online commented:
Доброй ночи, я не пойму, как правильно сдавать ДЗ.
Я постоянно делаю что то не так, как работать с гит я знаю, но не понимаю как лучше всего с ним работать. Как правильно сдавать ДЗ, что бы не трогать ваш код и что бы правильно делать Пулл реквест. А сейчас уже почти конец курса и я еще больше запутался, а после того раза как я у вас там что то заменил, еще больше начал переживать(

idoleg commented:
> Доброй ночи, я не пойму, как правильно сдавать ДЗ.
> Я постоянно делаю что то не так, как работать с гит я знаю, но не понимаю как лучше всего с ним работать. Как правильно сдавать ДЗ, что бы не трогать ваш код и что бы правильно делать Пулл реквест. А сейчас уже почти конец курса и я еще больше запутался, а после того раза как я у вас там что то заменил, еще больше начал переживать(

Вы сейчас учитесь, так что ничего нет страшного, я вас не осуждаю за то, что вы мой код чуть поправили :)
Просто вам нужно быть внимательнее и смотреть, какие, изменения попадают в ваш коммит.

idoleg reviewed (approved):
Без замечаний
